strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/nuget/NugetArtifactControllerTest.java

```diff
@@ -441,10 +432,10 @@ public void testLastVersionPackageSearch(@NugetRepository(storageId = STORAGE_ID
 
         // VERSION 2.0.0
         Path packagePathV2 = packagePaths.get(1);
-        packageContent = readPackageContent(packagePathV2);
+
         // Push
         url = getContextBaseUrl() + "/storages/{storageId}/{repositoryId}/";
-        createPushRequest(packageContent).when()
+        createPushRequest(packagePathV2).when()
                                          .put(url, storageId, repositoryId)
```

Comment:
Cosmetic: Could you please re-indent the chained methods? Thanks! :)

Reply:
Fixed.